getting_started.Rmd

```diff
@@ -74,80 +83,89 @@ In the pop up window, search for and add `jhudsl-robot`.
 
 Choose the `write` option then click `Add jhudsl-robot to this repository`.
 
+TODO: I don't see the write option.
```

Comment:
```suggestion
```

I think this only comes up for some types of repositories (like organizational ones not personal ones).